src/Plugins/LevelDBStore/IO/Data/LevelDB/DB.cs

```diff
@@ -37,12 +41,39 @@ public void Dispose()
             }
         }
 
+        /// <summary>
+        /// Remove the database entry (if any) for "key".
+        /// It is not an error if "key" did not exist in the database.
+        /// </summary>
+        public void Delete(byte[] key)
```

Comment:
"comments and style" adding new methods?

Reply:
i am already tying to help make it easier to review. but two or three lines of methods that is not being used or do anything, really? have to be sooooo strict to make it hard to work here? while people blame neo of not being good and is leaving neo ecosystem, we focus on making work harder and harder? how hard it is to review that few line methods? still a waste of your precious time?

Reply:
> "comments and style" adding new methods?

you as the guy who knows neo core better than anyone, i will always consider your comment and suggestion seriously, but if its not of any real help to the core but just blame, i am sorry i can not accept.